daml-script/dump/BUILD.bazel

```diff
@@ -0,0 +1,93 @@
+# Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+load(
+    "//bazel_tools:scala.bzl",
+    "da_scala_binary",
+    "da_scala_library",
+    "da_scala_test",
+)
+
+da_scala_binary(
+    name = "dump",
+    srcs = glob(["src/main/scala/**/*.scala"]),
+    main_class = "com.daml.script.dump.Main",
+    resources = glob(["src/main/resources/**/*"]),
+    scala_deps = [
+        "@maven//:com_github_scopt_scopt",
+        "@maven//:com_typesafe_akka_akka_stream",
+        "@maven//:org_scalaz_scalaz_core",
+        "@maven//:org_typelevel_paiges_core",
```

Comment:
Uh, what's this? 🙂 

EDIT: is it to compose the generated Daml Script?

Reply:
yep, the Doc stuff.